symphony-bdk-legacy/symphony-api-client-java/src/test/java/internal/FileHelperTest.java

```diff
@@ -29,4 +29,24 @@ public void should_read_file_from_system() throws IOException {
   public void fail_to_read_file() throws FileNotFoundException {
     FileHelper.readFile(UUID.randomUUID().toString() + ".abc");
   }
-}
\ No newline at end of file
+
```

Comment:
may I suggest a test with just classpath:, /file and classpath:, file
it looks it could be a corner case too

for this kind of test that are very simple and in the form of one input, one output I would sometime have a single test method and all the test cases in the same method
(but I don't mind having separated methods too :))